config/crds/serving_v1alpha1_knativeserving_crd.yaml

```diff
@@ -51,6 +51,19 @@ spec:
               description: A means to override the corresponding entries in the upstream
                 configmaps
               type: object
+            registry:
+              description: A means to override the registry for all knative images.
+              type: object
+              properties:
+                default:
+                  description: The default registry to use for all knative iamges.
+                    Takes the form of example-registry.io/custom/path/${NAME}:custom-tag
+                  type: string
+                override:
+                  description: The full image location of an individual knative service.
```

Comment:
I'm not sure you want to use the term "service" here. Maybe "container" if you follow my suggestion below?